design-system-templates/components/tr-search-results-family.html.twig

```diff
@@ -55,7 +55,7 @@
                 <dd>Michael Cooper</dd>
             </div>
             <div class="translation-list">
-                <dt>2 translations for Accessible Rich Internet Applications (WAI-ARIA) 1.0</dt>
+                <dt>Translations <span class="visuallyhidden">/span> Accessible Rich Internet Applications (WAI-ARIA) 1.0</span></dt>
```

Comment:
```suggestion
                <dt>Translations <span class="visuallyhidden">for Accessible Rich Internet Applications (WAI-ARIA) 1.0</span></dt>
```

Reply:
Apologies @vivienlacourba - I have pushed an amendment for that.